characters/css/index.css

```diff
@@ -766,7 +767,7 @@ input#burgerMenuState {
  * TABLES *
  **********/
 
-#mainTable th {
+ #mainTable th {
```

Comment:
space typo?

Reply:
this was a mistake, thanks for catching it